src/cobra/core/__init__.py

```diff
@@ -12,4 +12,3 @@
 from cobra.core.group import Group
```

Comment:
Would be nice to remove lines 1-4 i.e., `utf-8` encoding and `__future__` import.